Add `stopped` as a valid state for volume attachment

bodgit opened the pull request:
Should fix #1311

Ninir commented:
Hi @bodgit 

Do you think you would have time to finish this work and add the acceptance test asked by @radeksimko ?
Would be much appreciated! :)

bodgit commented:
I've added a test that passes:
```
$ make testacc TEST=./aws TESTARGS='-run=TestAccAWSVolumeAttachment_attachStopped'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -run=TestAccAWSVolumeAttachment_attachStopped -timeout 120m
=== RUN   TestAccAWSVolumeAttachment_attachStopped
--- PASS: TestAccAWSVolumeAttachment_attachStopped (199.85s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	199.880s
```
I also reverted my change and checked that the test also fails correctly.

jen20 commented:
Thanks for adding the test here - this looks good to me now!
